### ISSUE TITLE
Default profile not work in a fresh installation of Ubuntu 16.04.2

### ISSUE THREAD

**Issue by dpeite** [teltek/Galicaster#543]
This not happens when Ubuntu it's updated from 16.04 to 16.04.2

```
galicaster    2017-07-11 13:23:30,608    ERROR    recorder/recorder    gst-stream-error-quark: Internal data flow error. (1) (gstbasesrc.c(2948): gst_base_src_loop (): /GstPipeline:galicaster_recorder/galicaster+recorder+bins+videotest+GCvideotest:galicaster+recorder+bins+videotest+gcvideotest3/GstBin:bin4/GstVideoTestSrc:gc-videotest-src:
streaming task paused, reason not-negotiated (-4))
galicaster    2017-07-11 13:23:30,706    ERROR    recorder/service    Handle error (gst-stream-error-quark: Internal data flow error. (1) (gstbasesrc.c(2948): gst_base_src_loop (): /GstPipeline:galicaster_recorder/galicaster+recorder+bins+videotest+GCvideotest:galicaster+recorder+bins+videotest+gcvideotest3/GstBin:bin4/GstVideoTestSrc:gc-videotest-src:
streaming task paused, reason not-negotiated (-4)))
```

**Comment by dpeite** [comment]
This occurs because starting from Ubuntu 16.04.2 Ubuntu has [enabled HWE Stacks](https://wiki.ubuntu.com/Kernel/LTSEnablementStack). 

Fresh install of Ubuntu 16.04.2, use adaptor GLAMOR Textured Video, with has fewer video formats.
```bash
> xvinfo
X-Video Extension version 2.2
screen #0
  Adaptor #0: "GLAMOR Textured Video"
    number of ports: 16
    port base: 89
    operations supported: PutImage 
    supported visuals:
      depth 24, visualID 0x21
    number of attributes: 5
      "XV_BRIGHTNESS" (range -1000 to 1000)
              client settable attribute
              client gettable attribute (current value is 0)
      "XV_CONTRAST" (range -1000 to 1000)
              client settable attribute
              client gettable attribute (current value is 0)
      "XV_SATURATION" (range -1000 to 1000)
              client settable attribute
              client gettable attribute (current value is 0)
      "XV_HUE" (range -1000 to 1000)
              client settable attribute
              client gettable attribute (current value is 0)
      "XV_COLORSPACE" (range 0 to 1)
              client settable attribute
              client gettable attribute (current value is 0)
    maximum XvImage size: 8192 x 8192
    Number of image formats: 2
      id: 0x32315659 (YV12)
        guid: 59563132-0000-0010-8000-00aa00389b71
        bits per pixel: 12
        number of planes: 3
        type: YUV (planar)
      id: 0x30323449 (I420)
        guid: 49343230-0000-0010-8000-00aa00389b71
        bits per pixel: 12
        number of planes: 3
        type: YUV (planar)
```
Using Ubuntu 16.04.2 updated from 16.04:
```bash
> xvinfo
X-Video Extension version 2.2
screen #0
  Adaptor #0: "Intel(R) Textured Video"
    number of ports: 32
    port base: 75
    operations supported: PutImage 
    supported visuals:
      depth 24, visualID 0x20
    number of attributes: 1
      "XV_SYNC_TO_VBLANK" (range -1 to 1)
              client settable attribute
              client gettable attribute (current value is 1)
    maximum XvImage size: 16384 x 16384
    Number of image formats: 5
      id: 0x32595559 (YUY2)
        guid: 59555932-0000-0010-8000-00aa00389b71
        bits per pixel: 16
        number of planes: 1
        type: YUV (packed)
      id: 0x32315659 (YV12)
        guid: 59563132-0000-0010-8000-00aa00389b71
        bits per pixel: 12
        number of planes: 3
        type: YUV (planar)
      id: 0x30323449 (I420)
        guid: 49343230-0000-0010-8000-00aa00389b71
        bits per pixel: 12
        number of planes: 3
        type: YUV (planar)
      id: 0x59565955 (UYVY)
        guid: 55595659-0000-0010-8000-00aa00389b71
        bits per pixel: 16
        number of planes: 1
        type: YUV (packed)
      id: 0x434d5658 (XVMC)
        guid: 58564d43-0000-0010-8000-00aa00389b71
        bits per pixel: 12
        number of planes: 3
        type: YUV (planar)
  Adaptor #1: "Intel(R) Video Sprite"
    number of ports: 1
    port base: 107
    operations supported: PutImage 
    supported visuals:
      depth 24, visualID 0x20
    number of attributes: 2
      "XV_COLORKEY" (range 0 to 16777215)
              client settable attribute
              client gettable attribute (current value is 66046)
      "XV_ALWAYS_ON_TOP" (range 0 to 1)
              client settable attribute
              client gettable attribute (current value is 0)
    maximum XvImage size: 8192 x 8192
    Number of image formats: 3
      id: 0x32595559 (YUY2)
        guid: 59555932-0000-0010-8000-00aa00389b71
        bits per pixel: 16
        number of planes: 1
        type: YUV (packed)
      id: 0x59565955 (UYVY)
        guid: 55595659-0000-0010-8000-00aa00389b71
        bits per pixel: 16
        number of planes: 1
        type: YUV (packed)
      id: 0x18424752
        guid: 50415353-5448-524f-5547-485247423234
        bits per pixel: 32
        number of planes: 1
        type: RGB (packed)
        depth: 24
        red, green, blue masks: 0xff0000, 0xff00, 0xff
```

--------------------------------

Removing hwe packages and installing the correct packages fix the issue
```bash
sudo apt remove  xserver-xorg-core-hwe-16.04 #Also remove all the dependencies
sudo apt install ubuntu-desktop xorg xserver-xorg-core #Also install the required dependencies
```

**Comment by Alfro** [comment]
Added to docs: deb37239

**Comment by rubenrua** [comment]
Well done.

**Comment by ppettit** [comment]
i've struggled with this before and had to enable hwe to get newer hardware working - suggesting to remove it might cause issues?

**Comment by Alfro** [comment]
@ppettit In which hardware did you have these issues? Maybe we should consider adding a disclaimer with that.

Also, when we release the next bugfix version (2.0.1), removing hwe should not be necessary, so I think we could safely remove it from the docs at that point.

**Comment by dpeite** [comment]
Actually this bug it's fixed in the branch 2.0.x and will be in the next release 2.0.2 and not in 2.0.1 like @Alfro said.

**Comment by silviabf** [comment]
After a fresh installation of galicaster 2.0.2 removing hwe is not necessary, and default profile works correctly. I close this issue.
